torch/_inductor/scheduler.py

```diff
@@ -2145,6 +2193,21 @@ def add_user(
             for buf in node.get_outputs():
                 buf.set_users(name_to_users[buf.get_name()].items)
 
+        # compute and copy user information of donated buffer
```

Comment:
Nit: is there a reason to do this as a separate loop instead of where we are adding the other users above ?